configs/sst_accelerators-exclusion.yaml

```diff
@@ -0,0 +1,18 @@
+document: feedback-pipeline-unwanted
+version: 1
+data:
+  name: Accelerators Exclusion List
+  description: Accelerators packages not required in RHEL-9
+  maintainer: sst_accelerators
+  unwanted_packages:
+    - opae
+    - opae-devel
+    - spdlog
+    - spdlog-devel
```

Comment:
@raravind007 please note that recent (newer than c10s) oneapi-level-zero packages in ELN now require spdlog headers, and use `BuildRequires: spdlog-devel` to build with the system version rather than the bundled.  Therefore, for ELN (RHEL 11) and any possible rebases to RHEL 10, either spdlog will need to be in the buildroot, or oneapi-level-zero will need to be built with the bundled spdlog for RHEL (which will require a patch).